packages/tuf/src/cjson.js

```diff
@@ -0,0 +1,37 @@
+'use strict'
+/**
+ * Implementation of
+ * https://wiki.laptop.org/go/Canonical_JSON
+ * https://theupdateframework.github.io/specification/v1.0.30/index.html#metaformat
+ * that is needed for TUF serialization
+ */
+const { inspect } = require('util')
+
+function serialize (obj) {
```

Comment:
This function is now used nowhere except in tests (file `packages/tuf/test/cjson.spec.js`). It is better this way since I'd rather not run so much logic on data _before_ we validate the signature of said data.

(note: we don't need this function in `function checkIntegrity` because we get the data to verify using the much simpler `function extractSigned`, based on the assumption that the data sent by the backend is already canonicalised).

Since the only uses of this function in `packages/tuf/test/cjson.spec.js` are trivial, could we simply delete this file (so that we are not tempted to run this function on unvalidated data later on) and simply use already serialized data in `packages/tuf/test/cjson.spec.js`?